.gitignore

```diff
@@ -1,4 +1,5 @@
 .vs/*
 CemuStub/bin/*
 CemuStub/obj/*
-packages/*
\ No newline at end of file
+packages/*
+.vscode
```

Comment:
Again, including this to avoid VSCode files populating locally, since this editor isn't "officially" supported